var/spack/repos/builtin/packages/dmtcp/package.py

```diff
@@ -12,10 +12,12 @@ class Dmtcp(AutotoolsPackage):
     with no modifications to user code or to the O/S."""
 
     homepage = "http://dmtcp.sourceforge.net/"
-    url = "https://sourceforge.net/projects/dmtcp/files/2.6.0/dmtcp-2.6.0.tar.gz/download"
+    url = "https://github.com/dmtcp/dmtcp/archive/refs/tags/3.0.0.tar.gz"
```

Comment:
This project is a bit confusing. Their GitHub repo points to [sourceforge](https://dmtcp.sourceforge.io/) but appearntly the development of the tools has been moved to GitHub?

Reply:
Hi all,  Sorry for the delay in replying to this.  Please do reach out if there are further questions.  And I'll add a shout-out to @karya0 and @JainTwinkle (co-leaders of the DMTCP project), in case they have more to add.

At DMTCP, we are in the middle of moving the full support from Sourceforge to Github.  Eventually, all references to DMTCP will go through github.com.  But that's why you're currently sometimes seeing a mixture of sourceforge.com and github.com.

Please let us know if you have further questions.